pyanaconda/modules/subscription/subscription.py

```diff
@@ -81,6 +81,10 @@ def __init__(self):
         self._connect_to_insights = False
```

Comment:
A typo in the second commit:
``Due to this the is no setter...`` -> ``Due to this there is no setter...``

Reply:
Thanks! Fixed. :)